### ISSUE TITLE
[2.2.9] frameId not supported RemoteAgentError

### ISSUE THREAD

**Issue by garfield69** [FlareSolverr/FlareSolverr#536]
### Environment
* **FlareSolverr version**: 2.2.9
* **Last working FlareSolverr version**: 2.2.9 yesterday
* **Operating system**: win10
* **Are you using Docker**: [yes/no] no
* **FlareSolverr User-Agent (see log traces or / endpoint)**: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:94.0) Gecko/20100101 Firefox/94.0
* **Are you using a proxy or VPN?** [yes/no] no
* **Are you using Captcha Solver:** [yes/no] no
* **If using captcha solver, which one:**
* **URL to test this issue:** `https://torrentqq224.com/torrent/newest.html`

### Logged Error Messages
```
C:\ProgramData\flaresolverr>set LOG_LEVEL=debug

C:\ProgramData\flaresolverr>set LOG_HTML=true

C:\ProgramData\flaresolverr>flaresolverr
2022-10-04T14:16:00+13:00 INFO FlareSolverr v2.2.9
2022-10-04T14:16:00+13:00 DEBUG Debug log enabled
2022-10-04T14:16:00+13:00 INFO Testing web browser installation...
2022-10-04T14:16:00+13:00 DEBUG FlareSolverr user home directory is OK: C:\Users\Garfield69
2022-10-04T14:16:00+13:00 DEBUG Test URL: https://www.google.com
2022-10-04T14:16:00+13:00 DEBUG Creating new session...
2022-10-04T14:16:00+13:00 DEBUG Launching web browser...
2022-10-04T14:16:02+13:00 INFO FlareSolverr User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:94.0) Gecko/20100101 Firefox/94.0
2022-10-04T14:16:03+13:00 INFO Test successful
2022-10-04T14:16:03+13:00 INFO Listening on http://0.0.0.0:8191
2022-10-04T14:16:16+13:00 INFO REQ-1 Incoming request => POST /v1 body: {"maxTimeout":55000,"cmd":"request.get","url":"https://torrentqq224.com/torrent/newest.html"}
2022-10-04T14:16:16+13:00 DEBUG REQ-1 Creating new session...
2022-10-04T14:16:16+13:00 DEBUG REQ-1 Launching web browser...
2022-10-04T14:16:17+13:00 DEBUG REQ-1 Navigating to... https://torrentqq224.com/torrent/newest.html
2022-10-04T14:16:36+13:00 ERROR REQ-1 Unexpected error: ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1
navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13
execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25
execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25
execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20
onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36
onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18
handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14

2022-10-04T14:16:36+13:00 ERROR REQ-1 Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1
navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13
execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25
execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25
execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20
onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36
onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18
handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14

2022-10-04T14:16:36+13:00 INFO REQ-1 Response in 19.831 s
```

same for `https://bt4g.org/search/2022` , `https://idope.se/browse.html` , `https://torrentqq224.com/torrent/newest.html` , `https://zetorrents.nocensor.world/` , `https://torrentcore.xyz/index` , `https://pt.btschool.club/login.php`

**Comment by patosar** [comment]
Yeah, I've had this issue since about 5 hours, too.

**Comment by Hyperz** [comment]
Also seeing this issue since today.

**Comment by MrTyton** [comment]
+1, started happening last night.

**Comment by haohaoget** [comment]
+1

**Comment by mayziran** [comment]
+1

**Comment by pentago** [comment]
I'm guessing some 3rd party component got updated and caused this haywire..

**Comment by buginot** [comment]
Today:
 CAPTCHA challenge detected
2022-10-06T08:42:11+03:00 INFO REQ-2 Cloudflare Error: FlareSolverr can not resolve CAPTCHA challenges. Since the captcha doesn't always appear, you may have better luck with the next request.

Under any ip

**Comment by patosar** [comment]
> Today: CAPTCHA challenge detected 2022-10-06T08:42:11+03:00 INFO REQ-2 Cloudflare Error: FlareSolverr can not resolve CAPTCHA challenges. Since the captcha doesn't always appear, you may have better luck with the next request.
> 
> Under any ip

You ought to create a new issue, not comment here

**Comment by buginot** [comment]
I had a problem that in this thread has now turned into this one. I wrote here to check - have others also changed the situation?

**Comment by buginot** [comment]
> I had a problem that in this thread has now turned into this one. I wrote here to check - have others also changed the situation?



> > Today: CAPTCHA challenge detected 2022-10-06T08:42:11+03:00 INFO REQ-2 Cloudflare Error: FlareSolverr can not resolve CAPTCHA challenges. Since the captcha doesn't always appear, you may have better luck with the next request.
> > Under any ip
> 
> You ought to create a new issue, not comment here

I had a problem that in this thread has now turned into this one. I wrote here to check - have others also changed the situation?

**Comment by patosar** [comment]
> > I had a problem that in this thread has now turned into this one. I wrote here to check - have others also changed the situation?
> 
> > > Today: CAPTCHA challenge detected 2022-10-06T08:42:11+03:00 INFO REQ-2 Cloudflare Error: FlareSolverr can not resolve CAPTCHA challenges. Since the captcha doesn't always appear, you may have better luck with the next request.
> > > Under any ip
> > 
> > 
> > You ought to create a new issue, not comment here
> 
> I had a problem that in this thread has now turned into this one. I wrote here to check - have others also changed the situation?

I've just checked and this new behaviour hasn't happened for me.

**Comment by Cyrianu** [comment]
I'm seeing similar trying to download from fanfiction.net:

`2022-10-08T02:23:24-06:00 INFO REQ-10 Incoming request => POST /v1 body: {"cmd":"request.get","url":"https://www.fanfiction.net/s/9917094/1/","maxTimeout":60000,"cookies":[],"postData":null}
2022-10-08T02:23:47-06:00 ERROR REQ-10 Unexpected error: ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1
navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13
execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25
execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25
execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20
onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36
onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18
handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14

2022-10-08T02:23:47-06:00 ERROR REQ-10 Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5
UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1
navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13
execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25
execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25
execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20
onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36
onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18
handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14`

**Comment by ipaqi** [comment]
Just from looking online, the output looks similar to a puppeteer exception. Does it need to be fixed from that end, or is it localized to flaresolverr?

**Comment by caissonfiv** [comment]
I'm having the same problem with [m-team](https://kp.m-team.cc)

**Comment by buginot** [comment]
It looks like the problem won't be fixed soon. For many, all processes have stopped and there is no alternative

**Comment by garfield69** [comment]
As long as you are running a pretty standard Jackett and network, then there is a  beta version you could try, pre-built on docker, or source-built for windows using python3
https://github.com/FlareSolverr/FlareSolverr/issues/522#issuecomment-1257040575
But be aware it is **BETA**, so it does not yet have all the bells and whistles that compes with  the 2.2.9 release, so currently there is no socks support, no proxy support and no sessions support.
But it works well enough on my win10, and handles the remoteAgentError I reported.

**Comment by pentago** [comment]
Tried v3 on linux, didn't help.

**Comment by Cyrianu** [comment]
I'm using the fanficfare plugin for Calibre v6.6.1 on Win 10 Pro x64, and am trying to download from fanfiction.net.  With v2.2.9 I get the frameId not supported error per my previous comment [https://github.com/FlareSolverr/FlareSolverr/issues/536#issuecomment-1272264049].  With v3 beta installed using Python 3.10.7 I'm getting:

`2022-10-10 00:40:16 INFO     FlareSolverr 3.0.0.beta2
2022-10-10 00:40:16 INFO     Testing web browser installation...
2022-10-10 00:40:16 INFO     Chrome major version: 105
2022-10-10 00:40:20 INFO     FlareSolverr User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36
2022-10-10 00:40:20 INFO     Test successful
2022-10-10 00:40:20 INFO     Serving on http://0.0.0.0:8191
2022-10-10 00:41:40 INFO     Incoming request => POST /v1 body: {'cmd': 'request.get', 'url': 'https://www.fanfiction.net/s/9917094/1/', 'maxTimeout': 60000, 'cookies': [], 'postData': None}
2022-10-10 00:41:42 INFO     Challenge detected. Selector found: #trk_jschal_js
2022-10-10 00:42:42 ERROR    Error: Error solving the challenge. Timeout after 60.0 seconds.
2022-10-10 00:42:42 INFO     Response in 61.451 s
2022-10-10 00:42:42 INFO     127.0.0.1 POST http://localhost:8191/v1 500 Internal Server Error`

**Comment by garfield69** [comment]
can you specify the timeout for  flaresolverr in your fanficfare config? I have my Jackett config flaresolverr timeout set to 65000 (65 seconds)

**Comment by MrTyton** [comment]
Yes, that is possible to do. Default is currently set to 60 seconds, the
flag is `flaresolverr_proxy_timeout`

**Comment by Cyrianu** [comment]
I'm using the following settings in FanFicFare plugin for Calibre now (previously used the default of 60s timeout):

[www.fanfiction.net]
## fanfiction.net is blocking people more aggressively.  If you
## download fewer stories less often you can likely get by with
## reducing this sleep.
slow_down_sleep_time:20

## sites are sensitive to too many hits.  Users are sensitive to long
## waits during the initial metadata collection in the foreground.
## When used, these settings will speed up metadata downloads in the
## foreground linearly.
tweak_fg_sleep:true
min_fg_sleep:8.0
max_fg_sleep:20.0
max_fg_sleep_at_downloads:8

use_flaresolverr_proxy:true
# option settings, these are the defaults:
#flaresolverr_proxy_address:localhost
#flaresolverr_proxy_port:8191
#flaresolverr_proxy_protocol:http
flaresolverr_proxy_timeout:65000

Still getting the Challenge detected / Error solving the challenge on trying to use v3 beta, and getting frameId not supported with v2.2.9.

**Comment by Hyperz** [comment]
Haven't had much time to look closer into the issue but I've made some observations. In v2.2.9 the error is a result of the Puppeteer thinking the first request (https://github.com/FlareSolverr/FlareSolverr/blob/master/src/services/solver.ts#L140) never finished (timeout). But when you check what's going on without headless mode you can see the page did load and the challenge does get solved. My guess is that they changed the challenges added something to exploit this Puppeteer behavior. There have been sites that have done something similar before and the trick there is usually to make it not load a certain JS file. But this doesn't work with CF because of the nature of the challenges so I'm not sure what the solution here might be at this point. But I'm pretty sure it would have to be fixed on the Puppeteer side because Selenium doesn't have this issue.

In FlareSolverr v3 I think it's just the challenge detection that is broken (flaresolverr_service.py#182):
```python
challenge_found = False
for selector in CHALLENGE_SELECTORS:
    found_elements = driver.find_elements(By.CSS_SELECTOR, selector)
    if len(found_elements) > 0:
        challenge_found = True
        logging.info("Challenge detected. Selector found: " + selector)
        break
```

Replacing it with this solves the timeout issue until there's a proper fix:
```python
challenge_found = False
time.sleep(5)
```

I've also noticed the challenges solve significantly faster now than they used to.

**Comment by fede843** [comment]
+1 with this issue

**Comment by shubham399** [comment]
+1 with this issue

**Comment by ipaqi** [comment]
> Haven't had much time to look closer into the issue but I've made some observations. In v2.2.9 the error is a result of Puppeteer thinking the first request (https://github.com/FlareSolverr/FlareSolverr/blob/master/src/services/solver.ts#L140) never finished (timeout). But when you check what's going on without headless mode you can see the page did load and the challenge does get solved. My guess is that they changed the challenges, adding something to exploit this Puppeteer behavior. There have been sites that have done something similar before and the trick there is usually to make it not load a certain JS file. But this doesn't work with CF because of the nature of the challenges so I'm not sure what the solution here might be at this point. But I'm pretty sure it would have to be fixed on the Puppeteer side because Selenium doesn't have this issue.
> 
> In FlareSolverr v3 I think it's just the challenge detection that is broken (flaresolverr_service.py#182):
> 
> ```python
> challenge_found = False
> for selector in CHALLENGE_SELECTORS:
>     found_elements = driver.find_elements(By.CSS_SELECTOR, selector)
>     if len(found_elements) > 0:
>         challenge_found = True
>         logging.info("Challenge detected. Selector found: " + selector)
>         break
> ```
> 
> Replacing it with this solves the timeout issue until there's a proper fix:
> 
> ```python
> challenge_found = False
> time.sleep(5)
> ```
> 
> I've also noticed the challenges solve significantly faster now than they used to.

I don't get what you did here. you keep challenge_found as false, so it doesn't do anything. Can you explain how this works around the issue?

**Comment by Hyperz** [comment]
> I don't get what you did here. you keep challenge_found as false, so it doesn't do anything. Can you explain how this works around the issue?
> 
> edit: To clarify, I tried running this code with the time.sleep(5) both inserted before the for loop and completely replacing it, and even with completely replacing it and setting challenge_found to True. It still doesn't work.

All `challenge_found` being `True` does in that function is verify that the challenge was solved/completed by waiting until certain HTML elements are no longer present. Look at the code that follows:

```python
if challenge_found:
    while True:
        try:
            # then wait until all the selectors disappear
            for selector in CHALLENGE_SELECTORS:
                logging.debug("Waiting for selector: " + selector)
                WebDriverWait(driver, SHORT_TIMEOUT).until_not(
                    presence_of_element_located((By.CSS_SELECTOR, selector)))
            # all elements not found
            break
        except TimeoutException:
            logging.debug("Timeout waiting for selector")
            # update the html (cloudflare reloads the page every 5 s)
            html_element = driver.find_element(By.TAG_NAME, "html")
    # waits until cloudflare redirection ends
    logging.debug("Waiting for redirect")
    # noinspection PyBroadException
    try:
        WebDriverWait(driver, SHORT_TIMEOUT).until(staleness_of(html_element))
    except Exception:
        logging.debug("Timeout waiting for redirect")
    logging.info("Challenge solved!")
    res.message = "Challenge solved!"
else:
    logging.info("Challenge not detected!")
    res.message = "Challenge not detected!"

challenge_res = ChallengeResolutionResultT({})
challenge_res.url = driver.current_url
challenge_res.status = 200  # todo: fix, selenium not provides this info
challenge_res.cookies = driver.get_cookies()
```

`time.sleep(5)` waits long enough for the challenge to solve, redirect to the original URL. `challenge_found = False` bypasses all checks. The reason it works is because the `challenge_res` (and the solution cookies) gets set and returned regardless.

**Comment by ipaqi** [comment]
> > I don't get what you did here. you keep challenge_found as false, so it doesn't do anything. Can you explain how this works around the issue?
> > edit: To clarify, I tried running this code with the time.sleep(5) both inserted before the for loop and completely replacing it, and even with completely replacing it and setting challenge_found to True. It still doesn't work.
> 
> All `challenge_found` being `True` does in that function is verify that the challenge was solved/completed by waiting until certain HTML elements are no longer present. Look at the code that follows:
> 
> ```python
> if challenge_found:
>     while True:
>         try:
>             # then wait until all the selectors disappear
>             for selector in CHALLENGE_SELECTORS:
>                 logging.debug("Waiting for selector: " + selector)
>                 WebDriverWait(driver, SHORT_TIMEOUT).until_not(
>                     presence_of_element_located((By.CSS_SELECTOR, selector)))
>             # all elements not found
>             break
>         except TimeoutException:
>             logging.debug("Timeout waiting for selector")
>             # update the html (cloudflare reloads the page every 5 s)
>             html_element = driver.find_element(By.TAG_NAME, "html")
>     # waits until cloudflare redirection ends
>     logging.debug("Waiting for redirect")
>     # noinspection PyBroadException
>     try:
>         WebDriverWait(driver, SHORT_TIMEOUT).until(staleness_of(html_element))
>     except Exception:
>         logging.debug("Timeout waiting for redirect")
>     logging.info("Challenge solved!")
>     res.message = "Challenge solved!"
> else:
>     logging.info("Challenge not detected!")
>     res.message = "Challenge not detected!"
> 
> challenge_res = ChallengeResolutionResultT({})
> challenge_res.url = driver.current_url
> challenge_res.status = 200  # todo: fix, selenium not provides this info
> challenge_res.cookies = driver.get_cookies()
> ```
> 
> `time.sleep(5)` waits long enough for the challenge to solve and redirect to the original URL. `challenge_found = False` bypasses all checks. The reason it works is because the `challenge_res` (and the solution cookies) gets set and returned regardless.

Can confirm that v3 with this workaround works, at least in the limited circumstance I have been using it.

Since I use flaresolverr specifically for the FanFicFare addon for Calibre, I also had to contact the developer of that plugin and have them make the appropriate changes to their plugin to accomodate the changes in the response object in v3

**Comment by wjw054509** [comment]
等待解决方案

**Comment by f85386056** [comment]
FlareSolverr was unable to process the request, please check FlareSolverr logs. Message: Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5 UnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1 navigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13 execute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25 execute@chrome://remote/content/cdp/sessions/Session.jsm:64:25 execute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20 onPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36 onMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18 handleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14

**Comment by karanrahar** [comment]
Updating to 3.0.0.beta3 has solved this problem for me.

**Comment by alexreg** [comment]
@karanrahar Where do you find these beta releases? I see no such tags.

**Comment by karanrahar** [comment]
@alexreg 
https://github.com/FlareSolverr/FlareSolverr/issues/522
https://hub.docker.com/r/ngosang/flaresolverr/tags

**Comment by alexreg** [comment]
@karanrahar Thanks

**Comment by alexreg** [comment]
@laur89 Are you a maintainer? Care to elaborate?

**Comment by laur89** [comment]
I'm not, nothing to elaborate.

**Comment by alexreg** [comment]
> I'm not, nothing to elaborate.

What a freak haha.

**Comment by bientavu** [comment]
Hello, using beta v2 and v3, I'm also facing this error `Challenge detected. Selector found: #trk_jschal_js`. It is stuck then it times out. Hopefully a new version will fix this

**Comment by richarddan** [comment]
same here.

curl -L -X POST 'http://192.168.1.168:8191/v1' \                                                                                                                                                                ✔  04:25:49 PM
               -H 'Content-Type: application/json' \
               --data-raw '{
             "cmd": "request.get",
             "url":"https://kp.m-team.cc/takelogin.php",
             "maxTimeout": 60000}'
{"status":"error","message":"Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5\nUnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1\nnavigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13\nexecute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25\nexecute@chrome://remote/content/cdp/sessions/Session.jsm:64:25\nexecute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20\nonPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36\nonMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18\nhandleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14\n","startTimestamp":1668846350059,"endTimestamp":1668846355478,"version":"v2.2.10"}

**Comment by EviLex** [comment]
**And here is the same with last 3.0.0.beta3:**
curl -L -X POST 'http://127.0.0.1:8191/v1' \
-H 'Content-Type: application/json' \
--data-raw '{
  "cmd": "request.get",
  "url":"https://nnmclub.to/forum/login.php",
  "maxTimeout": 60000
}'
{"status":"error","message":"Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5\nUnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1\nnavigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13\nexecute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25\nexecute@chrome://remote/content/cdp/sessions/Session.jsm:64:25\nexecute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20\nonPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36\nonMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18\nhandleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14\n","startTimestamp":1669583526288,"endTimestamp":1669583549949,"version":"v2.2.10"}

**Comment by androiddisk** [comment]
I tried ghcr Io/flaresolver/flaresolver: late and ngosang/flaresolver: 3.0.0.beta3. I hope someone can solve it. thank you


```
curl -L -X POST 'http://localhost:8191/v1' -H 'Content-Type: application/json' --data-raw '{
  "cmd": "request.get",
  "url":"https://kp.m-team.cc/movie.php",
  "maxTimeout": 60000
}'
```

```
{"status":"error","message":"Error: Unable to process browser request. ProtocolError: Protocol error (Page.navigate): frameId not supported RemoteAgentError@chrome://remote/content/cdp/Error.jsm:29:5\nUnsupportedError@chrome://remote/content/cdp/Error.jsm:106:1\nnavigate@chrome://remote/content/cdp/domains/parent/Page.jsm:103:13\nexecute@chrome://remote/content/cdp/domains/DomainCache.jsm:101:25\nexecute@chrome://remote/content/cdp/sessions/Session.jsm:64:25\nexecute@chrome://remote/content/cdp/sessions/TabSession.jsm:67:20\nonPacket@chrome://remote/content/cdp/CDPConnection.jsm:248:36\nonMessage@chrome://remote/content/server/WebSocketTransport.jsm:89:18\nhandleEvent@chrome://remote/content/server/WebSocketTransport.jsm:71:14\n","startTimestamp":1670993493738,"endTimestamp":1670993515343,"version":"v2.2.10"}

```
```
{"status": "error", "message": "Error: Error solving the challenge. Timeout after 60.0 seconds.", "startTimestamp": 1670994910465, "endTimestamp": 1670994971205, "version": "3.0.0.beta3"}

```

**Comment by ngosang** [comment]
This error is fixed in FlareSolverr 3.0.0 => #650
v2 is no longer maintained.
